api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 246**
```

Comment:
You would also need to update the `API_FEATURE_LEVEL` field in `version.py`.

Reply:
thanks for letting me know! updated